pkgs/development/python-modules/pytest-pypi/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, six
+, flask
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-pypi";
```

Comment:
Not sure there's any value in adding this package, it's so unmaintained it's not even used by the pipenv tests which is presumably the intention of adding this in the first place.